packages/@storefront/flux-capacitor/src/core/actions/creators.ts

```diff
@@ -453,6 +453,21 @@ namespace ActionCreators {
     ];
   }
 
+  /**
+   * Selects the given refinements based on navigationId and indices.
+   * @param  {string}                               navigationId - The navigationId for
```

Comment:
Formatting is borked.

Reply:
Formatting is like this to keep the format the same with the rest of the file.

Reply:
Just marking that we discussed this *again* in IRL. Lets revisit normalizing the documentation blocks at some point in the future (ie. dropping 'space padding').